modules/machineset-efa-options.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: CONCEPT
+[id="machineset-efa-options_{context}"]
+= Machine sets that support using an Elastic Fabric Adapter
+
+You can use machine sets to create compute machines that use an link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa.html[Elastic Fabric Adapter (EFA)] as their primary network interface.
+
+[NOTE]
+====
+Control plane machines cannot use an EFA as their primary network interface.
+====
+
+The following AWS instance types support using an EFA:
+
+.Supported AWS instance types
+|====
+|Instance type |Sizes ^[1]^
+
+|General purpose
+a|* `m5dn.24xlarge`
+* `m5dn.metal`
+* `m5n.24xlarge`
+* `m5zn.12xlarge`
+* `m5zn.metal`
+* `m6a.48xlarge`
+* `m6a.metal`
+* `m6i.32xlarge`
+* `m6i.metal`
+* `m6id.32xlarge`
+* `m6id.metal`
+
+|Compute optimized
+a|* `c5n.18xlarge`
+* `c5n.9xlarge`
+* `c5n.metal`
+* `c6a.48xlarge`
+* `c6a.metal`
+* `c6gn.16xlarge`
+* `c6i.32xlarge`
+* `c6i.metal`
+* `c6id.32xlarge`
+* `c6id.metal`
+* `hpc6a.48xlarge`
+
+|Memory optimized
+a|* `r5dn.24xlarge`
+* `r5dn.metal`
+* `r5n.24xlarge`
+* `r5n.metal`
+* `r6i.32xlarge`
+* `r6i.metal`
+* `x2d.32xlarge`
+* `x2d.metal`
+
+* `x2ed.32xlarge`
+* `x2ed.metal`
+* `x2iezn.12xlarge`
+* `x2iezn.metal`
+
+|Storage optimized
+a|* `i3en.24xlarge`
+* `i3en.12xlarge`
+* `i3en.metal`
+* `i4i.32xlarge`
+* `i4i.metal`
+* `im4gn.16xlarge`
+
+|Accelerated computing
+a|* `dl1.24xlarge`
+* `g4dn.8xlarge`
+* `g4dn.12xlarge`
+* `g4dn.metal`
+* `g5.48xlarge`
+* `inf1.24xlarge`
+* `p3dn.24xlarge`
+* `p4d.24xlarge`
```

Comment:
These supported AWS instance types are referred from https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa.html "Supported instance types", so I don't think we need keep a copy here, as if aws changed that we should change this copy accordingly, I think we just need attach the link here, and we can combine this doc with modules/machineset-creating-efa-options.adoc together.

Reply:
Excellent suggestion, thank you! I will get this and all these other changes in soon.